devops/ansible/deploy-playground-playbook.yml

```
@@ -69,7 +69,7 @@
         TREASURY_ACCOUNT_URI: '{{ treasury_suri }}'
         INITIAL_BALANCES: '{{ initial_balances }}'
         SKIP_NODE: 'true'
-      async: 1800
+      async: 3600
```

Lets just add a little bit more time as we are just on the edge based on test runs I did.

```suggestion
      async: 4000
```